src/cnxml/deserialize.js

```diff
@@ -158,6 +184,7 @@ const mixed = type => (el, next) => ({
 const BLOCK_TAGS = {
     caption: caption,
     commentary: mixed('exercise_commentary'),
+    div: mixed('div'),
```

Comment:
What do we know about `div`s?

Reply:
You've added support for deserializing `div`s, but havn't included it in schema and don't serialize it.